common/log/tag/tags.go

```diff
@@ -927,3 +929,11 @@ func PartitionConfig(p map[string]string) Tag {
 func PollerGroups(pollers []string) Tag {
 	return newObjectTag("poller-isolation-groups", pollers)
 }
+
+func FallbackIsolationGroup(group string) Tag {
+	return newStringTag("fallback-isolation-group", group)
+}
+
+func PollerGroupsConfiguration(pollers types.IsolationGroupConfiguration) Tag {
```

Comment:
It's a duplicate of `PollerGroups`

Reply:
no, this is covering `types.IsolationGroupConfiguration` which is how most if the isolation groups are represented internally

Reply:
do you mind using a different key? "poller-isolation-groups" is used by another tag